### PR TITLE
feat: add player game length handler

### PR DESCRIPTION
**Pull request by godrien**
Description here: 
https://github.com/w3champions/website-backend/pull/281

### CODE REVIEW

File: W3ChampionsStatisticService/PlayerStats/GameLengthForPlayerStatistics/PlayerGameLengthStat.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+using System.Collections.Generic;
+using System.Linq;
+using W3ChampionsStatisticService.W3ChampionsStats.GameLengths;
+
+namespace W3ChampionsStatisticService.PlayerStats.GameLengthForPlayerStatistics;
+
+public class PlayerGameLengthStat
+{
+    // the dictionary key is the interval start: 0, 30, 60, 90, 120..
```
Comment (gustav87) [inline W3ChampionsStatisticService/PlayerStats/GameLengthForPlayerStatistics/PlayerGameLengthStat.cs:9]: Comment outdated?

File: W3ChampionsStatisticService/PlayerStats/GameLengthForPlayerStatistics/PlayerGameLengthStat.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    // the dictionary value is the number of games
+    public Dictionary<string, int> Lengths { get; set; }
+
+    public void Record(int duration)
+    {
+        var groupInterval = 60;
+        var maxGroupValue = 60;
+        var group = (int)duration / groupInterval;
```
Comment (gustav87) [inline W3ChampionsStatisticService/PlayerStats/GameLengthForPlayerStatistics/PlayerGameLengthStat.cs:17]: Why is a cast to int necessary?

Comment (godrien) [reply W3ChampionsStatisticService/PlayerStats/GameLengthForPlayerStatistics/PlayerGameLengthStat.cs:17]: To get a Integer instead of float

Comment (gustav87) [reply W3ChampionsStatisticService/PlayerStats/GameLengthForPlayerStatistics/PlayerGameLengthStat.cs:17]: Oh okay. I thought it was just casting the duration.

**Pull request merged**